src/elements/header/common/header-action.scss

```diff
@@ -37,6 +41,9 @@
   @include sbb.if-forced-colors {
     --sbb-header-action-border-color: CanvasText;
     --sbb-header-action-color: LinkText;
+
+    // Hide border with forced colors, as state is displayed on border
+    --sbb-header-action-active-border-width: 0rem;
```

Comment:
Is it possible to remove the 'rem' unit?

Reply:
yes. Will do.